examples/Gameraww/app/fetch.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function fetch(path) {
```

Comment:
I explicitly left it like that so that Gameraww can exercise both the ES6 modules and the CommonJS shim.